src/docs/compounds/image-with-text.hbs

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	{{> html-head title="Image With Text" }}
+</head>
+
+<body>
+	<div class="container mt-4">
+		<h1>Image With Text</h1>
+		<figure class="image-with-text mobile-wrapper row position-relative align-items-center mb-lg-2 mt-5">
+			<div class="col-12 col-lg-8 order-lg-last">
+				<div class="no-gutters__in-container">
+					<picture>
```

Comment:
can these be merged?

Reply:
I have tried, but couldn't make the image to be full width without that